docs/DeveloperGuide.md

```diff
@@ -453,40 +454,44 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
-
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Use the command `java -jar InSUREance.java` to launch the app
+   2. Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
need to include the `download the jar file and copy into an empty folder` part at the start

Reply:
i agree, thanks for pointing it out